Requirements for SSE.md

```diff
@@ -18,19 +18,21 @@
 
 #### TODO: Insert Use Case Name
 
-TODO: Insert use-case info
+Bob, a Home Assistant software user, recently purchased a Tesla and wants to know if it is possible to control his Tesla through Home Assistant. He notices that it is possible to lock your car, control climate, track sensor data, and track location. Home Assistant relies on Tesla’s API to access this information, which requires Bob’s Tesla account username and password. This feature can be configured via the GUI or manually. Bob chooses to manually configure the add-in since he is implementing some additional security features. Bob will be using a secrets.yaml file and the keyring, as recommended by the software documentation. 
 
 #### TODO: Misuse case Name
 
-TODO: Insert info here
+Bob goes to a get-together, Joe, a friend of a friend, is at the party. After Bob shows a party trick of being control and view his car from his home automation system, Joe notices that Bob uses Home Assistant. Joe has some cybersecurity knowledge and has used Home Assistant before. He knows that the software interfaces with a third-party API to gain the appropriate information, and the credentials are usually saved in the configuration file or a secrets.yaml file in plain text. As the night goes on, Joe scans the network and sees Bob has an SSH port forwarded and takes note of the WAN IP. After he goes home, he brute forces the simple password with a dictionary attack and gains complete access to Home Assistant because Bob did not disable root access via SSH. Joe looks over the documentation for the Tesla add-in and sees that it reports location. Before he goes any further and risk detection, he comes up with a plan to burglarize the home with some friends while Bob is at work. Joe wants to use the location services via a home assistant or gain access to the Tesla credentials to monitor Bob’s location without his knowledge. 
 
 #### Prevention/Security Requirement
 
-TODO: Assess alignment of security requirements derived from mis-use case analysis with advertised features of the open-source software. Review OSS project documentation and codebase to support your observations. Summarize your observations.
+Home Assistant recommends not allowing root access via SSH if forwarded to the public. 
+The documentation also recommends the user make sure secrets.yaml file to separate the configurations from the username and passwords. The documentation also recommends the use of a keyring to avoid storing passwords in plain text. Lastly, it is essential to secure SSH with a key rather than a password. 
```

Comment:
First sentence could maybe be --> The documentation also recommends the user to make sure that the secrets.yaml file separates the configurations from the username and passwords.